test/testsyncengine.cpp

```diff
@@ -381,7 +406,10 @@ private slots:
         QFETCH_GLOBAL(bool, filesAreDehydrated);
         QFETCH(bool, sameMtime);
         QFETCH(QByteArray, checksums);
-        QFETCH(int, expectedGET);
+        QFETCH(int, expectedGEThydrated);
+        QFETCH(int, expectedGETdehydrated);
+
+        int expectedGET = filesAreDehydrated ? expectedGETdehydrated : expectedGEThydrated;
```

Comment:
I expected the difference in the gets in a get triggered by 
`fakeFolder.localModifier().setContents(QStringLiteral("A/a1"), a1size, 'C');`
But this houls result in a constant increase of 1.
Also I added a compare that disproofs my assumption.
```cpp
fakeFolder.localModifier().setContents(QStringLiteral("A/a1"), a1size, 'C');
QCOMPARE(counter.nGET, filesAreDehydrated ? 1 : 0);```